server/src/api/modules/groups/group.utils.ts

```diff
@@ -16,4 +19,19 @@ function buildDefaultSocialLinks() {
   };
 }
 
-export { sanitizeName, buildDefaultSocialLinks };
+function sortMembers(group: Omit<GroupDetails, 'socialLinks' | 'memberCount'>): MembershipWithPlayer[] {
+  if (group.roleOrders && group.roleOrders.length) {
+    const roleOrderMap = new Map(group.roleOrders.map(r => [r.role, r.index]));
+    // this assumes roleOrders is sorted by index ascending out of the database
+    return [...group.memberships].sort(
+      (a, b) => (roleOrderMap.get(a.role) ?? 0) - (roleOrderMap.get(b.role) ?? 0)
```

Comment:
Doesn't defaulting to 0 mean any role that isn't in the order goes to the top?

Reply:
good call